### DISCUSSION TITLE
MIDI.turnThruOff();

### DISCUSSION THREAD

**Discussion by AaronHaCohen** [FortySevenEffects/arduino_midi_library#352] [Q&A]
MIDI.turnThruOff(); is documented and shown in the Chaining.ino example but that file won't compile. I'm trying to use it in the Callbacks.ino example, adding it to the setup at line 43. It compiles OK and uploads but MIDI thru is still not turned off. Any suggestions? Thanks
[Callbacks2.ino.txt](https://github.com/user-attachments/files/16563754/Callbacks2.ino.txt)

**Comment by lathoub** [comment]
#351
